common/functions/tests/it/scalars/maths/round.rs

```diff
@@ -0,0 +1,419 @@
+                ),
+                DataColumnWithField::new(
+                    DataColumn::Constant(DataValue::String(Some(b"aaa2".to_vec())), 1),
+                    DataField::new("d", DataType::String, false),
+                ),
+            ],
+            input_rows: 1,
+            func: RoundNumberFunction::try_create("round")?,
+            expect: Series::new([12346.0]).into(),
+            error: "",
+        },
+        Test {
+            name: "second arg is const string 'aaa'",
+            display: "round",
+            args: vec![
+                DataColumnWithField::new(
+                    Series::new([12345.6789]).into(),
+                    DataField::new("x", DataType::Float64, false),
+                ),
+                DataColumnWithField::new(
+                    DataColumn::Constant(DataValue::String(Some(b"aaa".to_vec())), 1),
+                    DataField::new("d", DataType::String, false),
+                ),
+            ],
+            input_rows: 1,
+            func: RoundNumberFunction::try_create("round")?,
+            expect: Series::new([12346.0]).into(),
+            error: "",
+        },
+        Test {
+            name: "second arg is 35",
+            display: "round",
+            args: vec![
+                DataColumnWithField::new(
+                    Series::new([
+                        0.12345678901234567890123456789012345,
+                        8888888888888888888888888888888888888888.0,
+                    ])
+                    .into(),
+                    DataField::new("x", DataType::Float64, false),
+                ),
+                DataColumnWithField::new(
+                    Series::new([35, 35]).into(),
+                    DataField::new("d", DataType::Int64, false),
+                ),
+            ],
+            input_rows: 2,
+            func: RoundNumberFunction::try_create("round")?,
+            expect: Series::new([
+                0.123456789012345678901234567890,
+                8888888888888888888888888888888888888888.0,
+            ])
+            .into(),
+            error: "",
+        },
+        Test {
+            name: "second arg is const 35",
+            display: "round",
+            args: vec![
+                DataColumnWithField::new(
+                    Series::new([
+                        0.12345678901234567890123456789012345,
+                        8888888888888888888888888888888888888888.0,
+                    ])
+                    .into(),
+                    DataField::new("x", DataType::Float64, false),
+                ),
+                DataColumnWithField::new(
+                    DataColumn::Constant(DataValue::Int64(Some(35)), 1),
+                    DataField::new("d", DataType::Int64, false),
+                ),
+            ],
+            input_rows: 2,
+            func: RoundNumberFunction::try_create("round")?,
+            expect: Series::new([
+                0.123456789012345678901234567890,
```

Comment:
this test works fine but when I run it from cmd sql I see some truncation in output:

```txt
mysql> SELECT ROUND(0.12345678901234567890123456789012345, 35);
+--------------------------------+
| ROUND(0.12345678901234568, 35) |
+--------------------------------+
|            0.12345678901234568 |
+--------------------------------+
```

Maybe it is related to just output...?